fix: pass snap type to obtain the snapcraft build plan (#119)
canonical/craft-platforms: docs/reference/changelog.rst

- Add a module of :doc:`/reference/testing/strategies` to assist when testing code that
  uses craft-platforms.

> Bug Fixes
> 
> - Properly pass snap type to obtain a Snapcraft build plan.

Reviewer: ```suggestion
Bug Fixes
=========

- Pass the correct type when obtaining a Snapcraft build plan.
```
